pkg/apis/core/types.go

```diff
@@ -2951,6 +2951,12 @@ const (
 	// DisruptionTarget indicates the pod is about to be terminated due to a
 	// disruption (such as preemption, eviction API or garbage-collection).
 	DisruptionTarget PodConditionType = "DisruptionTarget"
+	// PodResizePending indicates that the pod has been resized, but kubelet has not
+	// yet allocated the resources.
+	PodResizePending PodConditionType = "PodResizePending"
+	// PodResizing indicates that a resize is in progress, and is present whenever
+	// allocated resources != acknowledged resources.
+	PodResizing PodConditionType = "PodResizing"
```

Comment:
This was supposed to be `PodResizeInProgress` (https://github.com/kubernetes/enhancements/pull/5089#discussion_r1955038046) but it looks like I missed a whole bunch of places in the KEP when I renamed it. Sorry!

Reply:
updated